sound/soc/soc-dapm.c

```diff
@@ -3651,6 +3651,8 @@ snd_soc_dapm_new_control_unlocked(struct snd_soc_dapm_context *dapm,
 	case snd_soc_dapm_dac:
 	case snd_soc_dapm_aif_in:
 	case snd_soc_dapm_pga:
+	case snd_soc_dapm_buffer:
+	case snd_soc_dapm_scheduler:
```

Comment:
what about all the effects/src/asrc that were added at the same time as buffer/scheduler? Should they really be always on?

Reply:
@plbossart no, added them too now.